README.md

```diff
@@ -154,15 +150,15 @@ nebula-docker-compose/
         `- graph
 ```
 
-## Stopping the Services of **Nebula Graph**
+## Stop the **Nebula Graph** Services
 
-You can stop the services of **Nebula Graph** with the following command:
+You can stop the **Nebula Graph** services with the following command:
```

Comment:
If you want to stop ... , run the following command. 

the action word "run" is always recommended.